scripts/dump-schema.ts

```diff
@@ -13,16 +13,14 @@ if (destination === undefined) {
 }
 
 // Support both passing a folder or a filename
-const folder = path.dirname(destination)
-const filename = path.basename(destination) || "schema.graphql"
-if (!fs.existsSync(folder)) {
```

Comment:
I deleted this because if the user specifies a file in a base directory that doesn't exist, we'll get an error with a more helpful message when fs.writeFileSync fails.